azerothcore/azerothcore-wotlk
# fix(DB/Quest): Fixed Quest Stone Circle POIs

## Changes Proposed:
-  Fixed the missin POI from the quest map

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8347
- Closes https://github.com/chromiecraft/chromiecraft/issues/2017

## SOURCE:

https://tbc.wowhead.com/quest=3444/the-stone-circle

## Tests Performed:

- .quest add 3444
- Check map and see the icon on ratchet.
- Pick quest item. Check map and see the icon on Gazjetan

Test

Looking for the item

![stone circle 1](https://user-images.githubusercontent.com/87535580/136532408-61d26a37-7adf-4e8a-9aae-a264137f1c40.jpg)

After picking item

![image](https://user-images.githubusercontent.com/87535580/136532375-a4ebc87f-32c9-4d59-a658-a338eb867441.png)


## How to Test the Changes:


1. .quest add 3444
2. Check map and see the icon on ratchet.
3. Pick quest item. Check map and see the icon on Gazjetan



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
